README.md

```diff
@@ -40,20 +44,28 @@ The baseline for this dataset is Molecule Attention Transformer, fine-tuned with
 The baseline for this dataset is Audio Spectrogram Transformer, more details can be found in [paper](https://arxiv.org/abs/2104.01778).
 
 ## Question 1
-Can pretrained language models transfer to different modalities?
+* Can pretrained language models transfer to different modalities?
+* Only input embedding layer, output readout layer and layer norm parameters are fine-tuned.
+* FPT archives comparable performance to the fully trained transformer.
+* Models are learning representations and performing computation that is agnostic to the modality.
+
 
 ### Empirical results
 FPT trained with default parameters.
 
-// TODO
-
 Dataset | Metric Name | Result
 :---: | :---: | :---:
-Cyp3A4 Inhibition | Accuracy | ...
+Cyp3A4 Inhibition | Accuracy | ~70% (after 1k iterations)
+Speech Commands | Accuracy | ~10% (after 2,5k iterations)
+CIFAR10 | Accuracy | ~55% (after 2,5k iterations)
 
 ### Conclusions
 
-// TODO
+* It’s difficult to fully train a 12-layer transformer (gpt2-small) on (relatively small) datasets.
```

Comment:
We do not fully train a 12-layer transformer, but we rather fine-tune a frozen one. I suggest removing this line, because it reveals our technical inabilities.